### DISCUSSION TITLE
Easy way to use VITE_EXTRA_EXTENSIONS ?

### DISCUSSION THREAD

**Discussion by ms264556** [vuejs/vitepress#3571] [Q&A] [answered]
I have a few custom extensions which I wish to escape routing.

I see this is probably what I need: [feat: support specifying custom extensions to escape routing #3466](https://github.com/vuejs/vitepress/pull/3466).  
And this does work locally, if I add VITE_EXTRA_EXTENSIONS into an .env file.

Is there an easy & reliable way to add some extra extensions which works on all platforms? VITE_EXTRA_EXTENSIONS isn't working for me on Cloudflare Pages (which doesn't seem to honour the .env file nor use Setting > Environment Variables > VITE_EXTRA_EXTENSIONS ).

**Comment by brc-dd** [comment, marked as answer]
you can just do `process.env.VITE_EXTRA_EXTENSIONS = 'foo,bar'` on top of your vitepress config file

**Comment by ms264556** [comment]
Yes. This was the first thing I tried. But it seems to work very unreliably
when I navigate rather than hitting the page directly.

**Reply by brc-dd** [reply]
Which version are you on? I think it was not working as expected in one version in between. But directly overriding using process.env should work fine with latest release.

**Reply by ms264556** [reply]
Thank you!

I'd pulled your commit.  
I switched to the latest release and this works perfectly now.
